src/AutoRest.CSharp/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -19,8 +19,10 @@ public static async Task RunAsync(string[] args)
         {
             var basePath = args.Single(a=> !a.StartsWith("--"));
 
-            var configuration = LoadConfiguration(basePath, File.ReadAllText(Path.Combine(basePath, "Configuration.json")));
             var codeModelTask = Task.Run(() => CodeModelSerialization.DeserializeCodeModel(File.ReadAllText(Path.Combine(basePath, "CodeModel.yaml"))));
+            var codeModel = await codeModelTask;
+            var defaultLibraryName = codeModel.Language.Default.Name;
```

Comment:
Let's move this logic around a bit. Let's keep the `Configuration` for things we get directly from Autorest as parameters and have fallback logic in https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Output/Models/Types/BuildContext.cs#L27